src/GovUk.Education.ExploreEducationStatistics.Common/Model/Data/Query/ObservationQueryContext.cs

```diff
@@ -7,15 +8,18 @@ namespace GovUk.Education.ExploreEducationStatistics.Common.Model.Data.Query
     public class ObservationQueryContext
     {
         public Guid SubjectId { get; set; }
-        public TimePeriodQuery TimePeriod { get; set; }
-        public IEnumerable<Guid> Filters { get; set; }
+        public TimePeriodQuery? TimePeriod { get; set; }
+
+        public IEnumerable<Guid>? Filters { get; set; }
         // TODO EES-3328 - remove BoundaryLevel from ObservationQueryContext as we now store it on
         // MapChart configuration instead
         public long? BoundaryLevel { get; set; }
-        public IEnumerable<Guid> Indicators { get; set; }
+        public IEnumerable<Guid>? Indicators { get; set; }
         public List<Guid> LocationIds { get; set; } = new();
         [Obsolete("Legacy Location field that exists in queries of historical Permalinks", false)]
-        public LocationQuery Locations { get; set; }
+        public LocationQuery? Locations { get; set; }
+
+        public IncludeInResponseQuery? IncludeInResponse { get; set; }
```

Comment:
Think `ObservationQueryContext` is the wrong place to make these changes. In fact, `ObservationQueryContext` shouldn't be used for the filter subject meta endpoint at all really.

We should make a new request for filtering subject meta. This would only contain the things we're actually interested in e.g.

```cs
public record SubjectMetaFilterRequest
{
    public Guid SubjectId { get; set; }
    public TimePeriodQuery? TimePeriod { get; set; }
    public IEnumerable<Guid>? Filters { get; set; }
    public IEnumerable<Guid>? Indicators { get; set; }
    public List<Guid>? LocationIds { get; set; }
    public SubjectMetaFacetType IncludeFacetType { get; set; }
}

public enum SubjectMetaFacetType 
{
    Locations,
    TimePeriods,
    FilterItems,
    Indicators,
}
```

As mentioned in planning, I had issue with the `IncludeInResponse` stuff, so would suggest we change this to something simpler and more generic with the `IncludeFacetType` parameter.